ssl_cert_tracker/new_tasks.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+
+import logging
+import xml.dom.minidom
+
+from django.core.exceptions import ObjectDoesNotExist
+from celery import shared_task
+from libnmap.process import NmapProcess
+
+from ssl_cert_tracker.models import TestCertsData
+from orion_integration.lib import OrionSslNode
+
+logging.basicConfig(filename='pocProject.log',level=logging.DEBUG)
+
+@shared_task
+def go_node(node_id, node_address):
+    """Celery worker for each orion node"""
+    return_code = "Success"
+    net_map = NmapProcess(node_address, options='--script ssl-cert')
```

Comment:
net_map is a confuing name for an nmap run. you are not creating a map of the network, are you? please rename

Reply:
changed to nmap_task to industry standard